src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,8 +188,24 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resource-label [r language counts]
+  (let [organisation (get-in r [:organization :organization/short-name language])
+        duplicate? (> (get counts (:resid r)) 1)
+        licenses? (not-empty (:licenses r))]
+    (str (:resid r)
+         (when organisation
+           (str/join [" (" (text :t.administration/org) ": " organisation ")"]))
```

Comment:
`(str/join [x y z])` can be written more simply as `(str x y z)`

Reply:
oh, I didn't know that! Cool!

Reply:
I will not be refactoring this since it creates additional friction with testing, but overall it is a nice suggestion

Reply:
- replace nested whens with and
- remove string joins and replace with just one statement